### ISSUE TITLE
DN - Location Destination dan Source ID belum dinamis

### ISSUE THREAD

**Issue by lajayuhniyarsyah** [lajayuhniyarsyah/ERP-Supra#29]
@ekachandrasetiawan  : Di sbm_delivery_note.py

@metdod def create_picking(self, cr, uid, ids, context=None):

``` python
move_id = stock_move.create(cr,uid,{
                    'name' : x.name.name,
                    'origin':val.prepare_id.sale_id.name,
                    'product_uos_qty':x.qty,
                    'product_uom':x.product_uom.id,
                    'prodlot_id':x.prodlot_id.id,
                    'product_qty':x.qty,
                    'product_uos':x.product_uom.id,
                    'partner_id':val.partner_id.id,
                    'product_id':x.name.id,
                    'auto_validate':False,
                    'location_id' :12,
                    'company_id':1,
                    'picking_id': picking,
                    'state':'draft',
                    'location_dest_id' :9
                    },context=context)
```

location_dest_id = 9

9 == Customer Location

Customer Location punya xml_id = stock.stock_location_customers
- [ ] 

@ekachandrasetiawan  : Di method yang sama

``` python
'location_id' :12,
```

12 == HO
Di ubah menjadi sale.order.material.line.picking_location
- [ ]

**Comment by lajayuhniyarsyah** [comment]
Di method sama juga 

```python
'location_id' :12,
```

12 == HO
Di ubah menjadi sale.order.material.line.picking_location

**Comment by ekachandrasetiawan** [comment]
Penyesuaian Location 
https://github.com/ekachandrasetiawan/ERP-Supra/commit/3fc89fc20b0a74c3bf73fe5b26ca375aec2b1af1

**Comment by lajayuhniyarsyah** [comment]
@ekachandrasetiawan : untuk location_id cari dengan cara seperti link

http://stackoverflow.com/questions/8666809/how-to-get-the-database-id-from-an-xml-id#answer-12673957

**Comment by ekachandrasetiawan** [comment]
Penyesuaian XML ID location 

https://github.com/ekachandrasetiawan/ERP-Supra/commit/7672b4128074cc1ec873816deddab8326415c080

**Comment by lajayuhniyarsyah** [comment]
Merged #31
